src/main/app/controller/controller.utils.ts

```diff
@@ -0,0 +1,10 @@
+import { CaseWithId } from '../case/case';
+import { State } from '../case/definition';
+
+export const needsToExplainDelay = (userCase: Partial<CaseWithId>): boolean => {
```

Comment:
I tried this but it wasn't allowed because FinalisingYourApplicationPostController extends PostController, which requires instantiation. So when I tried using FinalisingYourApplicationPostController.needsToExplainDelay in the applicant1Sequence file, it threw an error.